swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -59,224 +55,229 @@ CREATE TABLE Company.Employees (
+
+3. Remove the generated content in the `main.bal` file and open the diagram view in VS Code.
+
+    ![Open diagram view](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/open-diagram-view.gif)
+
+## Create the data service
+
+The sections below demonstrate how to create the service.
 
 ### Create a record to represent an employee
 
-In Ballerina, records are a data type that maps keys to values. Define a closed record to represent a single row in the `Employees` table in the `main.bal` file. 
+In Ballerina, records are a data type that maps keys to values. Follow the steps below to define a closed record representing a single row in the `Employees` table in the `main.bal` file. 
 
 >**Info:** This record type is the basis for interacting with the database.
 
+Generate the record types corresponding to the payload from the service by providing the record name as `Employee` and the sample JSON object below.
+
+>**Tip:** You need to complete the generated record by adding the pipe signs to mark the record as a closed one, adding the `employee_id`, `manager_id`, and `hire_date` types, and importing the `ballerina/time` module, which cannot be represented in the JSON format.
+
+```json
+{    
+    "first_name": "test",
+    "last_name": "test",
+    "email": "test@test.com",
+    "phone": "882 771 110",
+    "job_title": "Sales Manager"
+}
+```
+
+![Create data record](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/create-data-record.gif)
+
+The completed record will be as follows.
+
 ```ballerina
 import ballerina/time;
 
-public type Employee record {|
-    int employee_id?;
+type Employee record {|
     string first_name;
     string last_name;
     string email;
     string phone;
-    time:Date hire_date;
-    int? manager_id;
     string job_title;
+    int employee_id?;
+    int? manager_id;
+    time:Date hire_date;
 |};
 ```
 
-### Add the MySQL driver
-
-The MySQL driver JAR is necessary to connect to and interact with a MySQL server. Select one out of the methods below to add it.
-
-1. Import the `ballerinax/mysql.driver` package in your `main.bal` file. This package bundles the latest MySQL driver so that the MySQL connector can be used in Ballerina packages easily.
-
-   ```ballerina
-   import ballerinax/mysql.driver as _;
-   ```
-   
-2. Update the `Ballerina.toml` file with the Maven dependency params for the MySQL driver.
-   ```toml
-   [[platform.java11.dependency]]
-   groupId = "mysql"
-   artifactId = "mysql-connector-java"
-   version = "8.0.26"
-   ```
-   
-3. Download the driver JAR manually and update the path in the `Ballerina.toml` file
-   ```toml
-   [[platform.java11.dependency]]
-   path = "/path/to/mysql/driver.jar”
-   ```
-   
 ### Define MySQL configurations
 
-In the package directory, create a new file named `Config.toml` and specify the configurations below to connect to the MySQL database.
+Follow the steps below to define the MySQL configurations.
```

Comment:
Instead of manually creating the config, use the tooling auto generated config feature during the run.

Reply:
Updated.